src/torchcodec/decoders/_core/FFMPEGCommon.cpp

```diff
@@ -7,6 +7,7 @@
 #include "src/torchcodec/decoders/_core/FFMPEGCommon.h"
 
 #include <c10/util/Exception.h>
+#include <iostream>
```

Comment:
Probably a remnant from debugging, although also not horrible to keep in.

Reply:
Good catch. Removed now.